plugins/arDominionB5Plugin/js/fullWidthTreeView.js

```diff
@@ -0,0 +1,320 @@
+"use strict";
+
+(function ($) {
+
+  $(function()
+  {
+    var $node = $('#treeview');
+
+    if ($node.length)
+    {
+      console.log("Loading treeview js");
+      $(loadTreeView);
+    }
+  });
```

Comment:
If this would be the only change, I'd consider it safe enough to do it in the existing JS file, but I see there is some Drupal bellow ...

Reply:
Sounds good - some additional changes with respect to deprecated jquery calls too. Tested in the old theme and functionality is still present.  Moving to the root /js folder.